docs/install/index.md

```diff
@@ -1,5 +1,133 @@
-There are a number of different methods to install and run Coder:
+1. Install Coder
+
+   <div class="tabs">
```

Comment:
Thoughts on changing to only 3 tabs?

1. Windows
2. macOS
3. Linux

and putting several options inside the tab contents. I ask because the tabs look a bit weird when scrollable


<img width="634" alt="Screenshot 2024-01-07 at 6 38 22 PM" src="https://github.com/coder/coder/assets/22407953/18d15c9f-21bf-4432-92a5-c339fac5ae8c">

Reply:
I agree that we should reduce the number of primary tabs. I will come up with a new organization. But please look at my comment above on why I do not want to group based on OS type.

Reply:
Updated to use three primary tabs.